sync/sync.go

```diff
@@ -15,6 +15,11 @@ import (
 
 var _ service.Service = (*Synchronizer)(nil)
 
+var (
+	StartingBlockNumber *uint64
+	HighestBlockHeader  *core.Header
+)
+
```

Comment:
Unless there is a real good reason, let's avoid package level variables. We can move these to `Synchronizer` struct.

Reply:
Done